src/web-ui/README.md

```diff
@@ -22,3 +22,46 @@ foo@bar:~$ docker-compose up --build web-ui
 ```
 
 Once the container is up and running, you can access it in your browser at [http://localhost:8080](http://localhost:8080).
+
+# Site Optimization with Moovweb XDN
+
+Read more information about XDN [here](https://developer.moovweb.com/guides/starter)
+
+## Links
+
+Project: https://moovweb.app/moovweb-demos/aws-rds
+
+Preview: https://moovweb-demos-aws-rds-default.moovweb-edge.io
+
+Carts AWS service proxied on XDN: https://github.com/moovweb-demos/aws-rds-carts-service/
+
+## Prerequisites
+
+XDN application works with Vue application production build files.
+To create it, follow the next steps.
+
+1. Make sure your terminal is open in `%project_root%/src/web-ui` folder
+2. Set NodeJS version >= 12 (to check your version use `node -v`)
+3. Install packages: run `npm install` (or just `npm i`)
+4. Set `VUE_APP_XDN_ENABLED` environment variable to `true` in `.env` file
+5. Build Vue application: `npm run build` (build files will appear in `dist` folder)
```

Comment:
This is just for local development and testing of the XDN, right?

The instructions for CI/CD using the CodePipeline configuration will be different, right? In other words these steps and the steps below will be done in CodePipeline and will not be necessary for the developer/user to run. Just want to make sure I understand the full vision for the integration so we can shape these instructions accordingly.

Reply:
They're basically the same everywhere but on CI/CD it's specific from project to project.
The build/deploy command sequence example for CI can look like this:
```bash
%% pre-instructions to configure nodejs version >= 12 (depends on CI pipeline APIs) %%
npm run build
npm run xdn:build
npx xdn deploy --skip-build --team=my-team --site=my-site-name
```
, where `--team` is the name of your team on https://moovweb.app and `--site` is the name of your site on it correspondingly